blog/themes/3mdeb/static/css/bulma.css

```diff
@@ -9711,9 +9711,7 @@ li.submenu {
   width: 200px;
 }
 ol > li {
```

Comment:
This changes also buttons in top menu. Perhaps separate selector for class="bigmenu"?

Reply:
I had some problems with generate localhost on mac. Created this css without have a look. Should works now.